docs/spec_draft.md

```diff
@@ -343,6 +344,102 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_inference
+
+### Semantics
+
+Normalizes the `operand` tensor across batch and spatial dimensions, using
+`mean` and `variance` provided in input, for each feature in the `feature_index`
+dimension and produces a `result` tensor. More formally, the semantics can be
+expressed using MLIR syntax as follows:
+
+```mlir
+func.func @upsize(%0: tensor<2xf32>) -> tensor<2x2x2xf32> {
+  %1 = mhlo.reshape %0 : (tensor<2xf32>) -> tensor<1x2x1xf32>
+  %2 = "mhlo.broadcast_in_dim"(%1){
+    broadcast_dimensions = dense<[0,1,2]> : tensor<3xi64>
+  } : (tensor<1x2x1xf32>) -> tensor<2x2x2xf32>
+  func.return %2 : tensor<2x2x2xf32>
+}
+
+func.func @batch_norm_inference(
```

Comment:
Can we check if https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/batchnorm_expander.cc gives the same behavior? 

Ignore if already checked.